ROADMAP.adoc

```diff
@@ -71,28 +71,41 @@ version name of a given release.
 
 [[minishift-140]]
 === Minishift 1.4.0
+- [x] Improve feedback to the user during startup  https://github.com/minishift/minishift/issues/485[#485]
 
-- [ ] Ability to create and manage multiple clusters https://github.com/minishift/minishift/issues/126[#126]
+[[minishift-160]]
+=== Minishift 1.6.0
+- [x] Ability to create and manage multiple clusters https://github.com/minishift/minishift/issues/126[#126]
 
-[[minishift-150]]
-=== Minishift 1.5.0
+[[minishift-170]]
+=== Minishift 1.7.0
+- [x] Assign Static IP address for Hyper-V https://github.com/minishift/minishift/issues/1316[#1316]
+- [x] Add support for Minikube ISO https://github.com/minishift/minishift/issues/1058[#1058]
 
-- [ ] Ability to manage users https://github.com/minishift/minishift/issues/390[#390]
-- [ ] Add a profile parameter https://github.com/minishift/minishift/issues/549[#549]
-- [ ] Provide caching of docker images https://github.com/minishift/minishift/issues/952[#952]
+[[minishift-180]]
+=== Minishift 1.8.0
+- [x] Add a minimum Openshift version for executing add-ons https://github.com/minishift/minishift/issues/1134[#1134]
 
-[[minishift-160]]
-=== Minishift 1.6.0
+[[minishift-190]]
+=== Minishift 1.9.0
+- [x] Provide notation for default value for addon-envs https://github.com/minishift/minishift/issues/1135[#1135]
+- [ ] Ability to use you own certificates for the cluster https://github.com/minishift/minishift/issues/391[#391]
```

Comment:
I don't think this is going to happen in this release. Even if started I have a hard time seeing it completed. IMO we might as well add it to 1.10.0